[18.0][MIG] stock_dangerous_goods: Migration to 18.0

xaviedoanhduy opened the pull request:
depend on: `https://github.com/OCA/community-data-files/pull/211`

pedrobaeza committed:
stock_picking_show_return

==============================
Show returns on stock pickings
==============================

This module enables one tab in the pickings that contains returns made for that
picking.

pedrobaeza committed:
stock_picking_show_return: Tests

pedrobaeza committed:
stock_picking_show_return: Migrated to 9.0

Commit:
stock_picking_show_return: Typo

The returned picking must be obtained from the picking of the iteration,
not the set of pickings to be calculated.

pedrobaeza committed:
[REN] stock_picking_show_returns > stock_picking_show_return

pedrobaeza committed:
[MIG] stock_picking_show_return: Migration to 10.0

oca-transbot committed:
OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

pedrobaeza committed:
[IMP] stock_*: Execute tests at the end

For avoiding errors when other modules add required fields on affected models

fmdl committed:
[MIG] Migrated stock_picking_show_return to v11.0

sbidoul committed:
remove obsolete .pot files [ci skip]

oca-travis committed:
[UPD] Update stock_picking_show_return.pot

sergio-teruel committed:
[11.0][ADD] stock_picking_auto_create_lot: Create lot for incoming pickings

sergio-teruel committed:
[11.0][ADD] stock_picking_scrap_quick: New module for scrap all picking products in one step

pedrobaeza committed:
[FIX] stock_picking_whole_scrap: Change module category

bizzappdev committed:
[MIG]stock_picking_show_return: migration from 11 to v12

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update stock_picking_show_return.pot

sergio-teruel committed:
[11.0][FIX] stock_picking_auto_create_lot: Not create lot if lot_name is stored in move_line

meigallodixital committed:
[MIG] stock_picking_auto_create_lot: Migration to 12.0

meigallodixital committed:
README generated with OCA README generator

meigallodixital committed:
FIX - Incorrect branch on README.rst and index.html

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-12.0/stock-logistics-workflow-12.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_picking_show_return/

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

pedrobaeza committed:
[IMP] stock_picking_auto_create_lot: Change latest comments:

- Promoted to Production/Stable
- Put maintainer
- Regenerate README
- Unify string
- Adapt translations accordingly

oca-travis committed:
[UPD] Update stock_picking_auto_create_lot.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-12.0/stock-logistics-workflow-12.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_picking_auto_create_lot/

sergio-teruel committed:
[MIG] stock_picking_whole_scrap: Migration to v12.0

ps-tubtim committed:
[IMP] stock_picking_auto_create_lot: black, isort

ps-tubtim committed:
[MIG] stock_picking_auto_create_lot: Migration to 13.0

oca-travis committed:
[UPD] Update stock_picking_auto_create_lot.pot

sergio-teruel committed:
[MIG] stock_picking_show_return: Migration to v13.0

sergio-teruel committed:
[IMP] stock_picking_show_return: black, isort

sergio-teruel committed:
[IMP] stock_picking_whole_scrap: black, isort

sergio-teruel committed:
[MIG] stock_picking_whole_scrap: Migration to v13.0

oca-travis committed:
[UPD] Update stock_picking_show_return.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_show_return/

CarlosRoca13 committed:
[ADD] stock_picking_return_restricted_qty New module

OCA-git-bot committed:
pre-commit update

OCA-git-bot committed:
pre-commit update

dong-z committed:
Added translation using Weblate (Chinese (Simplified))

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (4 of 4 strings)

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_auto_create_lot/zh_CN/

oca-travis committed:
[UPD] Update stock_picking_return_restricted_qty.pot

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

dong-z committed:
Added translation using Weblate (Chinese (Simplified))

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (5 of 5 strings)

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_show_return/zh_CN/

dong-z committed:
Added translation using Weblate (Chinese (Simplified))

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (3 of 3 strings)

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_return_restricted_qty/zh_CN/

dong-z committed:
Added translation using Weblate (Chinese (Simplified))

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_return_restricted_qty/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_show_return/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_return_restricted_qty/

rousseldenis committed:
[13.0][IMP] stock_picking_auto_create_lot: Improve performances and UX

Create lots using create_multi to reduce queries in case of large pickings.

Hide the serial assignement for products that are not enabled for it.

rousseldenis committed:
[13.0][IMP] stock_picking_auto_create_lot: Improve tests

Disable tracking, allows to inherit common tests values

sergio-teruel committed:
[ADD] stock_owner_restriction: New module to restrict quantities owned.
TT25269

rousseldenis committed:
[13.0][IMP] stock_picking_auto_create_lot: Allows to set lot on action_done

As the picking could be validated by other means than button, allows
to set lot anyway.

Improve tests

rousseldenis committed:
[13.0][IMP] stock_picking_auto_create_lot: Allow multi recordset

rousseldenis committed:
[13.0][IMP] stock_picking_auto_create_lot: Fix serial generation, add tests

ferran-S73 committed:
[14.0][MIG] stock_owner_restriction: Migration to 14.0

mmequignon committed:
Add stock_dangerous_goods

oca-travis committed:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
stock_picking_auto_create_lot 13.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-13.0/stock-logistics-workflow-13.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_auto_create_lot/

hugho-ad committed:
[IMP] stock_picking_show_return: black, isort, prettier

hugho-ad committed:
[MIG] stock_picking_show_return: Migration to 14.0

oca-travis committed:
[UPD] Update stock_picking_show_return.pot

OCA-git-bot committed:
[UPD] README.rst

sergio-teruel committed:
[FIX] stock_picking_show_return: Use Form in tests to avoid crash test due to this Odoo commit https://github.com/OCA/OCB/commit/8ca10a8f39c6e6b8647a695aa543f008f01ba034

sergio-teruel committed:
[FIX] stock_picking_return_restricted_qty: Use Form in tests to avoid crash test due to this Odoo commit https://github.com/OCA/OCB/commit/8ca10a8f39c6e6b8647a695aa543f008f01ba034

OCA-git-bot committed:
stock_picking_show_return 14.0.1.0.1

ValentinVinagre committed:
[MIG]stock_picking_auto_create_lot: Migration to 14.0

BernatPForgeFlow committed:
[IMP] stock_picking_show_return: black, isort, prettier

BernatPForgeFlow committed:
[MIG] stock_picking_show_return: Migration to 15.0

Commit:
[UPD] Update stock_picking_show_return.pot

OCA-git-bot committed:
[UPD] README.rst

mmequignon committed:
[MIG] stock_dangerous_goods: Migration to 14.0

pedrobaeza committed:
[IMP] Update dotfiles + switch to GH actions

CarlosRoca13 committed:
[MIG] stock_picking_whole_scrap: Migration to 15.0

Commit:
[UPD] Update stock_picking_whole_scrap.pot

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] stock_picking_whole_scrap: change deprecated tag on test

cesar-tecnativa committed:
[MIG] stock_owner_restriction: Migration to 15.0

Commit:
[IMP] stock_owner_restriction: documentation revision

CarlosRoca13 committed:
[IMP] stock_picking_return_restricted_qty: black, isort, prettier

CarlosRoca13 committed:
[MIG] stock_picking_return_restricted_qty: Migration to 15.0

Commit:
[UPD] Update stock_picking_return_restricted_qty.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[FIX]stock_picking_return_restricted_qty: fix quantity on return chain

Commit:
[IMP] stock_picking_auto_create_lot: pre-commit execution

Commit:
[MIG] stock_picking_auto_create_lot: Migration to 15.0

Commit:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
stock_picking_whole_scrap 15.0.1.0.1

ps-tubtim committed:
[MIG] stock_picking_auto_create_lot: Migration to 16.0

Commit:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_auto_create_lot/

ferran-S73 committed:
[15.0][IMP] stock_owner_restriction: Partner or unassigned stock option

rousseldenis committed:
[ADD] stock_picking_group_by_base

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 20.0% (1 of 5 strings)

Translation: stock-logistics-workflow-15.0/stock-logistics-workflow-15.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-15-0/stock-logistics-workflow-15-0-stock_picking_show_return/it/

victoralmau committed:
[FIX] stock_owner_restriction: Change SavepointCase to TransactionCase

OCA-git-bot committed:
[ADD] icon.png

AungKoKoLin1997 committed:
[MIG] stock_owner_restriction: Migration to 16.0

FranciscoFactorLibre committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_auto_create_lot/es/

slitheryduke11 committed:
[MIG] stock_picking_show_return: Migration to 16.0

Commit:
[UPD] Update stock_picking_show_return.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
stock_picking_show_return 16.0.1.0.1

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_show_return/

AungKoKoLin1997 committed:
[FIX] stock_picking_auto_create_lot: immediate transfer with auto_create_lot option

yajo committed:
[FIX] stock_owner_restriction: incompatibility with stock_barcode

When installing `stock_barcode` before installing this module, this one cannot be installed later.

@moduon MT-3005

rousseldenis committed:
[FIX] stock_picking_group_by_base: Avoid module load error due to DB lock timeout

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_show_return/es/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update stock_account_product_run_fifo_hook.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-15.0/stock-logistics-workflow-15.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-15-0/stock-logistics-workflow-15-0-stock_picking_return_restricted_qty/

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 83.3% (5 of 6 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_auto_create_lot/nl/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_owner_restriction
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_owner_restriction/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: stock-logistics-workflow-15.0/stock-logistics-workflow-15.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-15-0/stock-logistics-workflow-15-0-stock_picking_return_restricted_qty/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (29 of 29 strings)

Translation: stock-logistics-workflow-15.0/stock-logistics-workflow-15.0-stock_picking_whole_scrap
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-15-0/stock-logistics-workflow-15-0-stock_picking_whole_scrap/es/

josep-tecnativa committed:
[IMP] stock_move_line_auto_fill: test performance improvement
- Switch to setUpClass for avoiding repeat the same setup for each test.
- Include context keys for avoiding mail operations overhead.

josep-tecnativa committed:
[IMP] stock_owner_restriction: test performance improvement
- Include context keys for avoiding mail operations overhead.

josep-tecnativa committed:
[IMP] stock_picking_return_restricted_qty: test performance improvement
- Switch to setUpClass for avoiding repeat the same setup for each test.
- Include context keys for avoiding mail operations overhead.

josep-tecnativa committed:
[IMP] stock_picking_whole_scrap: test performance improvement
- Include context keys for avoiding mail operations overhead.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_account_product_run_fifo_hook
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_account_product_run_fifo_hook/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_show_return/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_auto_create_lot/it/

carlosdauden committed:
[IMP] stock_owner_restriction: Prevent a negative quant from being created with owner instead of reducing the original quant when a return is made by assigning owner.
Needed to avoid this restriction when packages are used:
https://github.com/odoo/odoo/blob/11d81b2145e95c50481101a63d3ad1d244279af4/addons/stock/models/stock_move.py#L1739
TT46054

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: stock-logistics-workflow-15.0/stock-logistics-workflow-15.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-15-0/stock-logistics-workflow-15-0-stock_picking_return_restricted_qty/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: stock-logistics-workflow-15.0/stock-logistics-workflow-15.0-stock_picking_whole_scrap
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-15-0/stock-logistics-workflow-15-0-stock_picking_whole_scrap/it/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_show_return/pt_BR/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_account_product_run_fifo_hook
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_account_product_run_fifo_hook/es/

Commit:
[MIG] stock_picking_show_return: Migration to 17.0

Commit:
[IMP] stock_picking_show_return: pre-commit auto fixes

Commit:
[UPD] Update stock_picking_show_return.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-17.0/stock-logistics-workflow-17.0-stock_picking_show_return
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-17-0/stock-logistics-workflow-17-0-stock_picking_show_return/

sergio-teruel committed:
[IMP] stock_owner_restriction: Add hook to allow to modify the owner behaviour by other modules

pedrobaeza committed:
[FIX] stock_picking*: Pre-commit fixes

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] stock_account_product_run_fifo_hook: Adapt to upstream changes

Product singleton after Odoo refactor

https://github.com/odoo/odoo/commit/670012a04f9d849ca8f0a42796ae47ecf138eaf8

ChrisOForgeFlow committed:
[16.0][FIX] stock_account_product_run_fifo_hook, update method to keep original algorithm in current version

sergio-teruel committed:
[IMP] stock_picking_auto_create_lot: Use Odoo core mechanism to create lots in pickings filling lot_name field instead of create all lots.

pedrobaeza committed:
[REM] stock_account_product_run_fifo_hook: Not usable right now

After the refactoring done in https://github.com/odoo/odoo/pull/164295

As the changes are very deep and out of the the PSC duties, it's
disabled for now for greenifying the branch.

ChrisOForgeFlow committed:
[FIX] added as installable stock_account_product_run_fifo_hook, stock_valuation_layer_usage

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_auto_create_lot/

AungKoKoLin1997 committed:
[IMP] stock_picking_auto_create_lot

Commit:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_auto_create_lot/

Commit:
[IMP] stock_picking_auto_create_lot: pre-commit execution

Commit:
[MIG] stock_picking_auto_create_lot: Migration to 17.0

Commit:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: stock-logistics-workflow-17.0/stock-logistics-workflow-17.0-stock_picking_auto_create_lot
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-17-0/stock-logistics-workflow-17-0-stock_picking_auto_create_lot/it/

JasminSForgeFlow committed:
[IMP] stock_account_product_run_fifo_hook: black, isort, prettier

JasminSForgeFlow committed:
[MIG] stock_account_product_run_fifo_hook: Migration to 17.0

Commit:
[UPD] Update stock_account_product_run_fifo_hook.pot

OCA-git-bot committed:
[BOT] post-merge updates

sergio-teruel committed:
[FIX] stock_owner_restriction: Wrong qty available computation for owner and product at the same time.

metaminux committed:
[FIX] stock_owner_restriction : apply standard behaviour if picking_type_id is not set on stock move

ced-adhoc committed:
[16.0][MIG] stock_picking_return_restricted_qty: Migration to 16.0

ced-adhoc committed:
[IMP] stock_picking_return_restricted_qty: add restrict_return_qty setting to stock.picking.type

Commit:
[UPD] Update stock_picking_return_restricted_qty.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_return_restricted_qty/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: stock-logistics-workflow-16.0/stock-logistics-workflow-16.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_return_restricted_qty/it/

mmrondon committed:
[MIG] sale_order_global_stock_route: Migration to 18.0

MiquelRForgeFlow committed:
[FIX] stock_account_product_run_fifo_hook: don't use taken_data as field name

Fixes https://github.com/OCA/stock-logistics-workflow/issues/1761

MiquelRForgeFlow committed:
[FIX] stock_account_product_run_fifo_hook: include last change

From https://github.com/odoo/odoo/commit/2f6f7fc8a3ea58cbe28a51776bfea49e4214b962.

Commit:
[UPD] Update stock_account_product_run_fifo_hook.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-17.0/stock-logistics-workflow-17.0-stock_account_product_run_fifo_hook
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-17-0/stock-logistics-workflow-17-0-stock_account_product_run_fifo_hook/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: stock-logistics-workflow-17.0/stock-logistics-workflow-17.0-stock_account_product_run_fifo_hook
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-17-0/stock-logistics-workflow-17-0-stock_account_product_run_fifo_hook/it/

mymage committed:
Added translation using Weblate (Italian)

cem-adhoc committed:
[MIG] stock_picking_auto_create_lot: Migration to 18.0

cem-adhoc committed:
[IMP] stock_picking_auto_create_lot: pre-commit execution

ced-adhoc committed:
[IMP] stock_picking_return_restricted_qty: pre-commit execution

ced-adhoc committed:
[17.0][MIG] stock_picking_return_restricted_qty: Migration to 17.0

xaviedoanhduy committed:
[IMP] stock_dangerous_goods: pre-commit auto fixes

xaviedoanhduy committed:
[MIG] stock_dangerous_goods: Migration to 18.0

chaule97 commented on stock_dangerous_goods/models/stock_quant_package.py:
```diff
@@ -0,0 +1,22 @@
+    def _has_lq_products(self):
+        self.ensure_one()
+        for quant in self.quant_ids:
+            if quant.product_id.is_lq_product:
+                return True
```
```suggestion
            return quant.product_id.is_lq_product
```

xaviedoanhduy replied:
why not return a boolean value directly, instead of having to scan the data again?

chaule97 reviewed (approved):
LGTM

Commit:
[UPD] Update stock_picking_return_restricted_qty.pot

OCA-git-bot committed:
[BOT] post-merge updates

ced-adhoc committed:
[18.0][MIG] stock_picking_return_restricted_qty: Migration to 18.0

JasminSForgeFlow committed:
[MIG] stock_account_product_run_fifo_hook: Migration to 18.0

OCA-git-bot committed:
Merge PR #1794 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update stock_account_product_run_fifo_hook.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_account_product_run_fifo_hook
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_account_product_run_fifo_hook/

BhaveshHeliconia committed:
[MIG] stock_picking_show_return: Migration to 18.0

OCA-git-bot committed:
Merge PR #1779 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update stock_picking_invoice_link.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_picking_invoice_link
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_picking_invoice_link/

OCA-git-bot committed:
Merge PR #1757 into 18.0

Signed-off-by rousseldenis

Commit:
[UPD] Update sale_order_global_stock_route.pot

OCA-git-bot committed:
[BOT] post-merge updates

natuan9 committed:
[IMP] stock_picking_group_by_base: pre-commit auto fixes

natuan9 committed:
[MIG] stock_picking_group_by_base: Migration to 18.0

juancarlosonate-tecnativa committed:
[IMP] stock_picking_whole_scrap: pre-commit auto fixes

juancarlosonate-tecnativa committed:
[MIG] stock_picking_whole_scrap: Migration to 18.0

TT54350

CarlosRoca13 committed:
[IMP] stock_owner_restriction: pre-commit auto fixes

CarlosRoca13 committed:
[MIG] stock_owner_restriction: Migration to 18.0

OCA-git-bot committed:
Merge PR #1841 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update stock_picking_whole_scrap.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_picking_whole_scrap
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_picking_whole_scrap/

OCA-git-bot committed:
Merge PR #1781 into 18.0

Signed-off-by rousseldenis

Commit:
[UPD] Update stock_picking_auto_create_lot.pot

OCA-git-bot committed:
[BOT] post-merge updates

bizzappdev committed:
[ADD] stock_dangerous_goods: add tests

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (31 of 31 strings)

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_picking_whole_scrap
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_picking_whole_scrap/it/

OCA-git-bot committed:
Merge PR #1814 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update stock_picking_show_return.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1845 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update stock_owner_restriction.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_owner_restriction
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_owner_restriction/it/

OCA-git-bot committed:
Merge PR #1790 into 18.0

Signed-off-by rousseldenis

Commit:
[UPD] Update stock_picking_return_restricted_qty.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_picking_return_restricted_qty
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_picking_return_restricted_qty/

OCA-git-bot committed:
Merge PR #1836 into 18.0

Signed-off-by jbaudoux

Commit:
[UPD] Update stock_picking_group_by_base.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_picking_group_by_base
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_picking_group_by_base/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-stock_picking_group_by_base
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-stock_picking_group_by_base/it/
